workspaces/acs/README.md

```diff
@@ -0,0 +1,118 @@
+# Advanced Cluster Security Backstage Plugin
+
+![ACS plugin image 1](images/acs_plugin_screenshot_1.png)
+![ACS plugin image 2](images/acs_plugin_screenshot_2.png)
+
+## For Administrators
+
+### Backstage Installation
+
+Run the following command to install the 3scale Backstage provider plugin:
+
+```console
+yarn workspace backend add @backstage-community/plugin-acs
+```
+
+### Configuration
+
+In `app-config.yaml` first add the proxies:
+
+```yaml
+proxy:
+  endpoints:
+    '/acs':
+      credentials: dangerously-allow-unauthenticated
```

Comment:
This should not be the default setting, this means that anyone can call this proxy endpoint. The default is `require` and that should be what we recommend as well. If an adopter changes this after the fact that's fine, they are making that choice. Most people will just copy paste so we want to lean toward the most secure option 👍 

More details here: https://backstage.io/docs/releases/v1.28.0/#breaking-proxy-backend-plugin-protected-by-default